.circleci/extended/s3-publish.yml

```diff
@@ -0,0 +1,39 @@
+# This config uses the `.circleci/extended/orb-test.yml` as its base, and then publishes the app to S3.
+# The workflow contained here demonstrates a practical application of `post-steps`
+# and using workspace persistence 
+version: 2.1
+
+orbs:
+  node: circleci/node@4.1
+  # The S3 orb is will setup the aws-cli and quickly copy or sync anything to s3.
+  # Orbs reduce the amount of configuration required for common tasks.
+  # See the orb documentation here: https://circleci.com/developer/orbs/orb/circleci/aws-s3
+  # NOTE: Environment variables containing the necessary secrets can be setup in the circleci UI
+  # See here https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project
+  aws-s3: circleci/aws-s3@2.0
+
+jobs:
+  s3-publish:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - attach_workspace:
+            at: /tmp/dist
+      - aws-s3/copy:
+          from: /tmp/dist
+          to: s3://sample-apps/vue
```

Comment:
@KyleTryon I tried creating this bucket using the creds for the s3-orb but it looks like it doesn't have permission to create. Do you have permission to? cc @CircleCI-Public/cpeng while Kyle's out

Reply:
Ill work on this with the new security policies.